docs/changelog.md

```diff
@@ -1,5 +1,283 @@
-v0.28 + 1
+v0.99
 ---------
```

Comment:
The script expects a newline between "----" and the body and complains if it doesn't find it. Initially, I've also had a check that verifies that section header and version are of equal lengths, but I've dropped that